config/testdata/conf.slack-default-api-url-file.yml

```diff
@@ -0,0 +1,22 @@
+global:
+  slack_api_url_file: '/global_file'
+
+route:
+  receiver: 'slack-notifications'
+  group_by: [alertname, datacenter, app]
+
+receivers:
+- name: 'slack-notifications'
+  slack_configs:
+  # Use global
+  - channel: '#alerts1'
+    text: 'test'
+  # Override global with other file
+  - channel: '#alerts2'
+    text: 'test'
+    api_url_file: '/override_file'
+  # Override global with inline URL
+  - channel: '#alerts3'
+    text: 'test'
+    api_url: 'http://mysecret.example.com/'
+    
```

Comment:
No newline at end of file. (But some whitespace on line 22, which just needs to be deleted?)